lib/models/StationContext.ts

```diff
@@ -0,0 +1,22 @@
+export class StationContext {
+    constructor(
+        private parentsPath: string[] = [],
+        private workingDirectory = '',
+    ) {}
+
+    getParentsPath() {
```

Comment:
Do we really need to have getters/setters ? Can we just mark fields in constructor as public